common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -2010,7 +2234,8 @@ object CelebornConf extends Logging {
       .createWithDefault(64)
 
   val WORKER_COMMIT_THREADS: ConfigEntry[Int] =
-    buildConf("celeborn.worker.commit.threads")
+    buildConf("celeborn.worker.shuffle.commit.threads")
```

Comment:
I think the origin is better or ```celeborn.worker.commitFiles.threads```. Maybe we should make a principle that there shouldn't be ```shuffle``` inside a config name.

Reply:
yea